docs/getting-started/edfi-exchange/technology/epp-candidate-employment-starter-visuals.mdx

```diff
@@ -0,0 +1,39 @@
+<img src="https://edfidocs.blob.core.windows.net/$web/img/edfi-exchange/technology/DSM%20logo%20(1).png" width="300" />
+
+## Description
+
+This project created two proof of concepts to have Ed-Fi ODS data visualized for Educator Preparation Programs designed by a group of Texas EPPs. The two proof of concepts were:
+
+* EPP Candidate Employment and Placement
+* EPP Candidate Employment and Retention
+
+ To confirm that EPPs are meeting the needs of K-12 LEAs by preparing a diverse corps of teachers ready to increase student learning and support high-needs schools, EPPs need to know where, what and whom their former teacher candidates are teaching, and if they remain in the profession. The visuals allow the users to track changes in positions/roles/locations of their candidates.
+
+ An existing Ed-Fi ODS connection is required to be able to visualize data in a similar way. Rather than a finalized “production” dashboard that’s ready to be used for decision making, this dashboard should instead be properly QA’d with data validation before use. These templates could be used as stepping stones for another EPP looking to visualize data in PowerBI. This version is a prototype rather than a finished product.
+
+## Download
+
+Proof of Concept #1 - **Candidates Employment and Placement**
+
+* [Power Bi Dashboard](https://edfi.atlassian.net/wiki/download/attachments/22496627/POC%20%231%20PowerBI.pbix?version=1&modificationDate=1676500956853&cacheVersion=1&api=v2)
+* [Functional Documentation](https://edfi.atlassian.net/wiki/download/attachments/22496627/Functional%20Documents%20EPP%20PoC%231.zip?version=2&modificationDate=1676588029713&cacheVersion=1&api=v2) (zip file containing Microsoft Word, Excel and PDF files)
+
+Proof of Concept #2 - **Candidates Employment and Retention**
+
+* **Code:** [https://github.com/Ed-Fi-Exchange-OSS/Ed-Fi-EPP-Candidate-Employment-PowerBI-Dashboard](https://github.com/Ed-Fi-Exchange-OSS/Ed-Fi-EPP-Candidate-Employment-PowerBI-Dashboard)
+* [Functional Documentation](https://edfi.atlassian.net/wiki/download/attachments/22496627/Functional%20Documents%20EPP%20PoC%232.zip?version=1&modificationDate=1676501446300&cacheVersion=1&api=v2) (zip file containing Microsoft Word files)
+* [Technical Documentation](https://edfi.atlassian.net/wiki/download/attachments/22496627/POC%20%232%20Technical%20Documents.zip?version=1&modificationDate=1676501547407&cacheVersion=1&api=v2) (zip file containing Microsoft Word, Excel and PDF files)
+
+## Details
+
+* **By:**  NewLeef, LLC and DSM Technical Solutions Inc. with a grant from Bill & Melinda Gates Foundation
+* **License terms:** Apache-2.0
+* **Released:**  2023
+
+## **At a Glance**
+
+**Generation****:** Tech Suite 3
+**For:** ODS/API ![(question)](https://edfi.atlassian.net/wiki/s/695013191/6452/be943731e17d7f4a2b01aa3e67b9f29c0529a211/_/images/icons/emoticons/help_16.png)
```

Comment:
I think we didn't know which version of the ODS/API this supported. Let's replace the image with "5.x". This is a reasonable assumption for something developed in 2023, especially given that EPP programs had not moved to 6.x yet.